go.mod

```diff
@@ -1,6 +1,8 @@
 module helm.sh/helm/v3
 
-go 1.22.0
+go 1.22.4
+
+toolchain go1.23.4
```

Comment:
Can the `toolchain` directive be removed?

For additional clarity for reviewers, oras v1.2.6 requires go 1.22.4

Reply:
Ack, done. 👍🏻

Reply:
I was wondering if oras-go should come out with a 1.2.7 that requires 1.23.0 or even 1.22.0 to avoid this

Reply:
> I was wondering if oras-go should come out with a 1.2.7 that requires 1.23.0 or even 1.22.0 to avoid this

Does `oras-go` actually need go `1.22.4` (or higher)? It would be better IMHO for `oras-go` to specify the minimum go version only. And rely on module consumers to build with a new enough go tool chain for e.g. CVE purposes.

Reply:
Seems https://github.com/oras-project/oras-go/pull/791/files#r1680475454 is the problem